src/pages/workspaces/index.tsx

```diff
@@ -0,0 +1,129 @@
+import { useState, useEffect } from "react";
+import Header from "../../components/header";
+import Focus from "../../components/workspaces/focus";
+import { Main, Footer } from "../../components/custom-styles";
+import { FcBar, FcHr } from "../../components/workspaces/focus.style";
+import BoardsList from "../../components/workspaces/boards-list";
+import { MiniBdList } from "../../components/workspaces/boards-list.style";
+
+const drelloBoardsList = [
+  {
+    id: 1,
+    title: "Dimsum",
+    focus: true,
+    workspace: {
+      id: "wk2",
+      name: "Family",
+      type: "Engineering - IT",
+      description: "",
+    },
+    stats: {
+      milestones: ["50% Completed", "23% In Progress", "27% Backlog"],
+      team_members: ["eddydev", "syndrome_123"],
+      card_summary: [
+        "34 total issues",
+        "18 closed",
+        "12 dropped",
+        "1 archived",
+      ],
+    },
+  },
+  {
+    id: 2,
+    title: "Project Echo",
+    focus: true,
+    workspace: {
+      id: "wk2",
+      name: "Family",
+      type: "Engineering - IT",
+      description: "",
+    },
+    stats: {
+      milestones: ["50% Completed", "23% In Progress", "27% Backlog"],
+      team_members: ["eddydev", "syndrome_123"],
+      card_summary: [
+        "34 total issues",
+        "18 closed",
+        "12 dropped",
+        "1 archived",
+      ],
+    },
+  },
+  {
+    id: 3,
+    title: "Drello",
+    focus: false,
+    workspace: {
+      id: "wk1",
+      name: "Setunas",
+      type: "Engineering - IT",
+      description: "",
+    },
+    stats: {
+      milestones: ["50% Completed", "23% In Progress", "27% Backlog"],
+      team_members: ["eddydev", "syndrome_123"],
+      card_summary: [
+        "34 total issues",
+        "18 closed",
+        "12 dropped",
+        "1 archived",
+      ],
+    },
+  },
+];
+
+const Workspaces = () => {
+  const [focusView, setFocusView] = useState(true);
+  const focusBoardList = drelloBoardsList.filter((item) => item.focus === true);
+
+  const toggleFocus = () => {
+    const focusToggle = !focusView;
+    setFocusView(focusToggle);
+  };
+
+  return (
+    <>
+      <Header title="Drello" />
+      <Main>
+        {focusView ? (
+          <>
+            <FcBar onClick={() => toggleFocus()}>
```

Comment:
Just FYI, you can also write like this 😀 Staying your way is also totally OK.
```suggestion
            <FcBar onClick={toggleFocus}>
```